interpreter/script/run.ml

```diff
@@ -375,6 +375,14 @@ let assert_result at got expect =
             let open Values in
             let open Simd in
             match shape, v with
+            | I32x4, V128 v ->
+                    let l0 = I32 (V128.I32x4.extract_lane 0 v) in
```

Comment:
Nit: odd indentation?

Reply:
Yea, might as well fix the indentation below too!